spec/controllers/revisions_controller_spec.rb

```diff
@@ -47,6 +55,13 @@
       end
     end
 
+    it 'does not return revisions that happened after the basic course ended' do
```

Comment:
`does not return revisions from the final day of the basic course but after it ended`